src/read_write/input_file_iterator.rs

```diff
@@ -21,12 +21,10 @@ use std::path::PathBuf;
 #[derive(Debug)]
 pub enum InputFile {
     Ply(PathBuf),
-    Pts(PathBuf),
 }
 
 pub enum InputFileIterator {
     Ply(PlyIterator),
-    Pts(PtsIterator),
 }
 
 impl Iterator for InputFileIterator {
```

Comment:
seems like InputFileIterator is now unnessecary.

Reply:
True, I removed it, too, PTAL!